apidoc/Titanium/UI/UI.yml

```diff
@@ -2807,6 +2807,15 @@ properties:
     osver: {ios: {min: "13.0"}}
     since: "9.1.0"
 
+  - name: cutoutSize
+    summary: Position and shape of the Android notch
+    description: |
+        Using Android 9+ it will return the bounding box of the Android notch. It will return `top`,`left` and `width`, `height`.
+    type: Number
```

Comment:
The type is not a number IIRC. You can define an own type, e.g.
```
---
name: CutoutSize
extends: Padding
summary: Dictionary object of parameters for the <Titanium.UI.cutoutSize>.
since: "12.1.0"
platforms: [android]
properties:
  - name: left
    type: Number
    summary: The left margin of the cutout.

  - name: right
    type: Number
    summary: The left margin of the cutout.

  - name: width
    type: Number
    summary: The width of the cutout

  - name: height
    type: Number
    summary: The height of the cutout
```

And then apply it with:
```
...
type: CutoutSize
...
```